### ISSUE TITLE
optirun and wine problems

### ISSUE THREAD

**Issue by bambulk0** [Bumblebee-Project/Bumblebee#711]
I'm running optirun on my laptop with wine for years now without a problem. However, if I use a wine version above 1.6, it doesn't work together with optirun. wine without optirun works, and optirun without wine also works.

[backtrace.txt](https://github.com/Bumblebee-Project/Bumblebee/files/56652/backtrace.txt)

**Comment by ArchangeGabriel** [comment]
Well, that’s weird. However, curious system where you have such an old kernel with such a recent wine. You should probably try upgrading your distro (which is ?)…

**Comment by bambulk0** [comment]
I'm using Ubuntu 14.04. That doesn't seem that old to me.

**Comment by ArchangeGabriel** [comment]
Indeed, your distro is a recent LTS, so you might have good reason using it. However wine 1.8 is not even two weeks old, so what I’m pointing here is a huge disparity in freshness among the software components you use (and that is not recommended). ;)

Did it worked with previous version of wine (like the 1.6.2 one shipped in Ubuntu)? As your previous log is about 1.8rc3, did you test this with final 1.8?

**Comment by bambulk0** [comment]
It worked in 1.6, but not in 1.7. Also the final 1.8 doesn't help.

**Comment by ArchangeGabriel** [comment]
OK, so at this point we have two options: either it’s an issue with wine 1.8 or with the fact you use a version of wine too recent for your system. What you could do is try a live version of a recent distro with persistence, install and configure bumblebee + wine 1.8 within it and see if it works.

**Comment by ArchangeGabriel** [comment]
What the status here?

**Comment by tdm4** [comment]
Hi, I get a similar message from WINE 1.8.2 (in Debian Jessie backports) using optirun.
Usually the message manifests itself in "The program encountered a problem and needs to close".
On the command-line I get:

` wine: Unhandled page fault on write access to 0x7bf02fe8 at address 0x7d6802ea (thread 002e), starting debugger...`

with no further output

**Comment by tdm4** [comment]
I tried adding `WINEDEBUG=+relay` and couldn't find where it died amongst the reams and reams of output
